examples/test/local/test_bert_cuda_gpu.py

```diff
@@ -16,14 +16,14 @@ def setup():
 
 
 @pytest.mark.skip(reason="Disable failing tests")
-@pytest.mark.parametrize("search_algorithm", ["tpe"])
+@pytest.mark.parametrize("sampler", ["tpe"])
 @pytest.mark.parametrize("execution_order", ["joint", "pass-by-pass"])
 @pytest.mark.parametrize("system", ["local_system"])
 @pytest.mark.parametrize("olive_json", ["bert_cuda_gpu.json"])
```

Comment:
This test will fail if you updated the config file name. We can use `bert.py` you added for this test.

Reply:
Fixed